uint8 mode #This sets the operating mode to one of the OI_MODE states from the TurtlebotSensorState.msg 
---
bool valid_mode # will return false if a unvalid mode was requested